client/src/components/Slider/Slider.css

```diff
@@ -0,0 +1,48 @@
+.slide {
+  height: 280px;
+  width: 260px;
+  margin-right: 30px;
+}
+
+.title {
+  width: 290px;
+  background-color: rgb(246, 230, 222);
+  text-align: left;
+  font-weight: bold;
+}
+
+.card {
+  display: flex;
+}
+
+.slide-container {
+  display: flex;
+  flex-direction: row;
+  height: 260px;
+  width: 300px;
+  background-color: rgb(246, 230, 222);
+  margin-right: 0;
+}
+
+.button {
+  background-color: rgb(246, 230, 222);
+  border: 0;
+  margin: 0;
+  font-size: 20px;
+  height: 278px;
+}
+
+.slide-img {
+  height: 210px;
+  width: 125px;
+  margin: 10px;
+  margin-bottom: 0;
+}
+
+.slide-text {
```

Comment:
Also update the font here!

Reply:
I am not there yet as I am having a problem using the fonts that are imported for the project (the font needed is there).  I assure you I am working rigorously to remedy that and will keep you posted.